docs/host_install.md

```diff
@@ -0,0 +1,189 @@
+`PDO_SOURCE_ROOT` points to the PDO source directory. Information about
+other [environment variables](environment.md) that control build and
+installation is described below.
+
+## Process Overview
+
+- Setup the basic development environment
+- Download TinyScheme source
+- Install SGX SDK
+- Install SGX SSL
+- Install required build dependencies
+- Set up environment variables to configure the build
+- Build and install Sawtooth
+- Build the PDO package
+
+## <a name="environment">Setup Build Environment</a>
+
+On a minimal Ubuntu system, the following packages are required. Other
+distributions will require similar packages.
+
+```bash
+sudo apt install -y cmake curl git pkg-config unzip xxd libssl-dev
+sudo apt install -y swig python3 python3-dev python3-venv virtualenv
+sudo apt install -y liblmdb-dev libprotobuf-dev libsecp256k1-dev protobuf-compiler
+```
+
+<!--
+    virtualenv will cause python3 and python3-virtualenv to be installed
+-->
+
+## <a name="tinyscheme">Build Tinyscheme</a>
+
+This project contains a modified version of the Tinyscheme interpreter
+for use within a secure enclave.  You **also** need a separate, plain
+copy of Tinyscheme to use outside the enclave for contract development.
+
+- Download the Tinyscheme source:
+```bash
+wget https://downloads.sourceforge.net/project/tinyscheme/tinyscheme/tinyscheme-1.41/tinyscheme-1.41.zip -P /tmp
+```
+
+- Extract and compile it:
+```bash
+cd ${PDO_SOURCE_ROOT}
+unzip /tmp/tinyscheme-1.41.zip
+cd tinyscheme-1.41
+make FEATURES='-DUSE_DL=1 -DUSE_PLIST=1'
+```
+
+- Set the `TINY_SCHEME_SRC` environment variable to the directory where
+you built the package (this environment variable will be used in
+the PDO build process so you might consider adding it to your login
+shell script (`~/.bashrc` or similar):
+
+```bash
+export TINY_SCHEME_SRC=${PDO_SOURCE_ROOT}/tinyscheme-1.41
+```
+
+## Install the SGX SDK
+
+Private Data Objects has been tested with version 2.4 of the SGX
+SDK. You can download prebuilt binaries for the SDK and kernel drivers
+from https://01.org/intel-software-guard-extensions/downloads.
+
+The following commands will download and install version 2.4 of the SGX
+SDK. When asked for the installation directory, we suggest that you install
+the SDK into the directory `/opt/intel`.
+
+```bash
+wget https://download.01.org/intel-sgx/linux-2.4/ubuntu18.04-server/sgx_linux_x64_sdk_2.4.100.48163.bin -P /tmp
+sudo /bin/bash /tmp/sgx_linux_x64_sdk_2.4.100.48163.bin
+```
+
+The installer includes a file that sets environment variables to
+configure the SGX SDK installation. For example, if you installed the
+SDK into the directory `/opt/intel`, then the following will set the
+necessary SGX environment variables:
+
+```bash
+source /opt/intel/sgxsdk/environment
+```
+
+You can also build and install the SGX SDK from source. Instructions for
+building from source are available
+[Intel SGX SDK git repository](https://github.com/intel/linux-sgx).
+
```

Comment:
Is there any reason to build and install the SGX SDK from source? If not, it might be preferable to avoid this line

Reply:
I believe this to be a helpful reference.